project/AkkaDisciplinePlugin.scala

```diff
@@ -38,7 +38,8 @@ object AkkaDisciplinePlugin extends AutoPlugin {
     "akka-persistence-typed",
     // references to deprecated PARSER fields in generated message formats?
     "akka-persistence-query",
-    "akka-docs")
+    "akka-docs",
+    "akka-stream-tests")
```

Comment:
I have to add this to let the build pass.

Reply:
Fine but would be good with a comment explaining why like for the other ones